### DISCUSSION TITLE
Add a quick action button in the navigation

### DISCUSSION THREAD

**Discussion by taliesinpenbardd** [filamentphp/filament#15197] [Help]
### Package

Panel builder

### Package Version

v3.2.131

### How can we help you?

Hey all,

Pretty new to Filament, but I have already a functionnal backend, and I'm in awe for your work so far. 

What I'd like, right now, is to know if there is a way to add a quick create button in the navigation, instead of the [badge](https://filamentphp.com/docs/3.x/panels/navigation#adding-a-badge-to-a-navigation-item) that can appear in regard of it ? I've read how to add a [custom navigation item](https://filamentphp.com/docs/3.x/panels/navigation#registering-custom-navigation-items), but that adds a link in the first level of the navigation.

Anyway, I'm on my way to convert my clients to Laravel instead of WordPress! :)

**Comment by leandrocfe** [comment]
Maybe you could use a [render hook](https://filamentphp.com/docs/3.x/support/render-hooks#overview) to achieve what you want

**Reply by taliesinpenbardd** [reply]
That's a good suggestion, but I tried without any luck. I've searched through [this chart](https://filamentexamples.com/tutorial/render-hooks-cheat-sheet) and with the documentation you mentioned, but wasn't able to make it happen...

Any other idea, by any chance?  😃
